documentation/modules/ostack-token-auth.adoc

```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: PROCEDURE
+[id="openstack-token-authentication_{context}"]
+= Using token authentication with an {osp} source provider
+
+You can use token authentication instead of username/password authentication when you use {project-first} to migrate a virtual machine that has an {osp} source provider.
+
+{project-short} supports both of the following types of token authentication:
+
+* Token with user ID
+* Token with user name
+
+For each type of token authentication, you need to use data from OpenStack to create a `Secret` manifest.
+
+.Prerequisites
+
+Have an {osp} account.
+
+.Procedure
+
+. In the dashboard of the {osp} web console, click *Project > API Access*.
+. Expand *Download OpenStack RC file* and click *OpenStack RC file*.
++
+The file that is downloaded, referred to here as `<openstack_rc_file>` includes the following fields used for token authentication:
++
+[source, terminal]
+----
+OS_AUTH_URL=http://192.168.100.181:5000
+OS_PROJECT_ID=dbed38bf449b495295868afa61a565dc
+OS_PROJECT_NAME="forklift"
+OS_DOMAIN_NAME="Default"
+OS_USERNAME="forklift"
+----
+
+. To get the data needed for token authentication, run the following command:
++
+[source,terminal]
+----
+$ openstack token issue
+----
++
+The output, referred to here as `<openstack_token_output>`, includes:
++
+* The `token` `userID`, and `projectID` you need for authentication using a token with user ID (`projectID` is the sme as `OS_PROJECT_ID` in `<openstack_rc_file>` as ).
+* The `username` and `projectName` you need for authentication using a token with user name (`username` in the output is the same as `OS_USERNAME`in `<openstack_rc_file>``). You also must use `OS_DOMAIN_NAME` from `<openstack_rc_file>`if you use this method.
+
```

Comment:
@anarnold97 Same question as before, but no need to repeat your answer.

Reply:
@RichardHoch - changed